apps/allocations/app/components/App/App.js

```diff
@@ -128,7 +127,7 @@ const App = () => {
       <Header
         primary="Allocations"
         secondary={
-          <Button mode="strong" icon={<IconPlus />} onClick={onNewAccount} label="New Account" />
+          <Button mode="strong" icon={<IconPlus />} onClick={onNewBudget} label="New Account" />
```

Comment:
If we want consistency, we should want to also rename the label
```suggestion
          <Button mode="strong" icon={<IconPlus />} onClick={onNewBudget} label="New budget" />
```

Reply:
Just click on "Commit suggestion" to apply my code change automatically :)